### PR TITLE
Bumps android gradle plugin version to work with AS 2.3

### PR DESCRIPTION
**Pull request by rharter**

### CODE REVIEW

File: integration/app/build.gradle [hunk 1]
```diff
@@ -33,5 +33,5 @@ dexcount {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.2.1'
+    compile 'com.android.support:appcompat-v7:25.1.0'
```
Comment (jaredsburrows) [inline integration/app/build.gradle]: Can we bump to `25.2.0`?

File: integration/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (jaredsburrows) [inline integration/gradle/wrapper/gradle-wrapper.properties]: Can we bump to `3.4`?

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (jaredsburrows) [inline gradle/wrapper/gradle-wrapper.properties]: Can we bump to `3.4`?

**Comment by jaredsburrows** [comment]
@rharter TravisCI takes forever to fix things sometimes, we just need to update the `.travis.yml` file to this:

```yaml
install:
  - echo y | android update sdk -u -a -t tools
  - echo y | android update sdk -u -a -t platform-tools
  - echo y | android update sdk -u -a -t build-tools-25.0.2
  - echo y | android update sdk -u -a -t android-25
  - echo y | android update sdk -u -a -t extra-google-m2repository
  - echo y | android update sdk -u -a -t extra-android-m2repository
```

https://github.com/travis-ci/travis-ci/issues/6555#issuecomment-282635361

**Comment by rharter** [comment]
@jaredsburrows Good call.  There you go.

**Comment by jaredsburrows** [comment]
@rharter Thanks but the build will still fail because of the TravisCI issue.

**Comment by jaredsburrows** [comment]
Replace this(https://github.com/KeepSafe/dexcount-gradle-plugin/blob/master/.travis.yml#L3) with my comment. Then it should pass!

**Comment by rharter** [comment]
Ahh, misread that as just to versions in the yaml file.  I'll update that now.

File: .travis.yml [hunk 4]
```diff
@@ -17,7 +17,13 @@ groovy:
 jdk:
   - oraclejdk8
 
-install: true
+install:
```
Comment (rharter) [inline .travis.yml]: Is this all that's needed?  How does that effect the `components` section?

**Comment by jaredsburrows** [comment]
@rharter Just delete `android: components` and use the `install:` only. Like this:

.travis.yml
```yaml
language: android

jdk:
  - oraclejdk8

install:
  - echo y | android update sdk -u -a -t tools
  - echo y | android update sdk -u -a -t platform-tools
  - echo y | android update sdk -u -a -t build-tools-25.0.2
  - echo y | android update sdk -u -a -t android-25
  - echo y | android update sdk -u -a -t extra-google-m2repository
  - echo y | android update sdk -u -a -t extra-android-m2repository

script:
  - ./.buildscript/test.sh

after_success:
  - ./.buildscript/deploy_snapshot.sh

env:
  global:
    - secure: jt1Zcx55jOaZZIbTRdLfe89Uoof3WdHyNylETQzl13M/AdXBRrnmNPiC9nRJaa0thdAbRjtqTCcZ5bkP3eCEywIJ3kaLSUoPzbccH0BMUFFdXBPqwGgY2+XdDJumFPeu/Mg4LyGMN3pzwaOkIql+Wm12up/16P0AfAF8NuA1aSL8Vslmh93tU8Rm0W7jsm1KapaWhH1JF8smKu5UX325GSMpROpze7P0uVz0iqdtqhx4kRtNsMLdwFNaXzovWWIQ4gdoQqVr9tyo3I6o9jptPftcIlqtQoS3S0xqD9ekyh8WlTUkBj0zBn1RKbNI8v4n9qvZ6i4IwL3T446jcjwA2sUrMtdUS+YDVfqJT5PHa7ojaaXg/k0YZ9YyATVY6kjxofDYOAmkU97B+cN7T5Mv42huUjsRyEv8M5xoSm3ls2qgaiAmlXxL/DtFuzXdBTbEkgGu2N1GdgvlOhquzO9IzKQZ8JoJiUUQ9Tl5NxXSjZU3SoRMazw04zN40Q6C/gYJ1hV23VGq7bUQN2we5TSBejOEMgn6ND8oPyKSdqReA+ekwd2EjCYacqTgyvCkWg+dZKPUXbK9lUxuQploiBuOY7WJSv4ANnfeJoxbMx9lydB3ks/J4skuEjc9v20WO74xavsSm7eANJ28xayxqWg3cPJCPd/XVDCt49Hqo0VfZ28=
    - secure: Y3UuQXQ6+NQQa7RMp7hf88Biv5VfcPmdz/y1hPaNWg/Ux5xKV9E3CqTCbjJs+dZP8xKo/DLARnDQxUEiBTzmAXOvCGinI9iYgW2zRohhKWWKQ9a/O2cIevXgOJ5pIMTCAEac68ysQpHRw6MtSg/Ha3rGDr0INtghDkvmv4Kn55fz7lun7Pp0koCufyZl1/NE0BgOS556Px4ZbPH4PpdNuD86Xxg2NJam26dPLu8jFvC4JxPfdylQEsR/fBmQtGRk9GLQOqYKH2scA3sipWTBI6SAMHVJScte5FKcnClump8My7NjrZCMT97rk7iETN97/IxWe2yVdFEqQOptYWzUMUJT3AppqtlvMxm5kTzPt5kLY2Xl8AITvOW3BCkmaTa4x3dQtu+fNf4jyFONSOuv/x4O0oTLOC3C8E5BmsGJsp+rok2Nbv6yFF3etj9ENH54jO9cpo4xFfDeSuvqi9pP5CsBQHzVt9Vi/3KafK+a1JvauskICI3+CcyKXLDZho2VNFFdRE/mqRcFBXa2wr7tzs7ic1P9nVRisv/PxmHBQeNsnYyiJUUVIh3jFN6M0VZPnRkE20YLwdWs7jqlsL+voipGkqSn1TGYj4wCt7H/+f6Om2mdnmcdrRVZ2kMyW6E0kILHdSiygPbEaaXpKZckQwr+Vku+zMck/OjvvpQRqqc=

sudo: false

cache:
  directories:
    - $HOME/.gradle
```

**Comment by jaredsburrows** [comment]
@rharter O!
I think the method count changed because newer versions?

```
Total time: 1 mins 21.724 secs
Building integration test project...
Integration build done!  Running tests...
Incorrect method count in app-debug.apk
The command "./.buildscript/test.sh" exited with 1.
```

**Comment by jaredsburrows** [comment]
LGTM.

**Comment by benjamin-bader** [comment]
Thanks to the both of you!

**Pull request merged**